madgraph/iolibs/files.py

```diff
@@ -117,7 +117,7 @@ def is_uptodate(picklefile, path_list=None, min_time=1343682423):
     
     assert type(path_list) == list, 'is_update expect a list of files'
     
-    pickle_date = os.path.getctime(picklefile)
+    pickle_date = os.path.getmtime(picklefile) # FIX BUG mg5amcnlo/mg5amcnlo#89 (use getmtime instead of getctime)
```

Comment:
```suggestion
    pickle_date = os.path.getctime(picklefile)
```

Do not think that this is needed here.

Reply:
Ok you are right, sorry for the noise and mistake and thanks for cross-checking. I will undo this.